bin/license.py

```diff
@@ -102,14 +108,18 @@
     "Eclipse Public License": [],
     "Public Domain License": [],
     "Mozilla Public License Version 2.0": [],
-    "Unicode License": [],
+    "Unicode/ICU License": [],
     "Go License": [],
     "Other License": []
 }
 
 for _ in licenses:
+    # Because the license of this project itself dose not need to be declared here
     if "org.apache.seatunnel" in _:
         continue
+    # Because the license is not in compliance, if you need to use MySQL, you can add it yourself
```

Comment:
I noticed it, that’s why I added this comment

Reply:
@CalvinKirs here

Reply:
However, the license of MySQL dependence has been reported by license-maven plugin, so the script have to ignore it

Reply:
You should check your code, normally it shouldn't make such an error. He is not our dependency.

Reply:
No, this is not about code

Reply:
Like I said before, it caused by license-maven-plugin with sortArtifactByName option

Reply:
So does this mean that this tool is not available and that it is likely to lead us into a trap?

Reply:
No I don’t think so

Reply:
For me, the license is very important, I can't accept such an error (even if its probability is small),@kezhenxu94 WDYT?

Reply:
Maybe it would better that you can merge this first and go through from the begin to understand the whole logic, and then we can discuss more effectively.

Reply:
This issue is not resolved and I hope that this issue gets discussed by others so that we can reach a consensus.

Reply:
I agree the license is important, but I disagree there is error.

Reply:
Maybe it would better that you can merge this first and go through from the begin to understand the whole logic, and then we can discuss more effectively.

Reply:
You just solved the current problem, when if there is a second one similar to MySQL (protocol compatible), but we don't actually use it, this situation is a potential problem for me

Reply:
The script can handle it, and mark the Github Action job as failed, and then we can know and avoid to use it. Really, you should read the script and understand the logic first please.

Reply:
Then can you tell me why the problem of ignoring mysql occurs?
https://github.com/apache/incubator-seatunnel/pull/1387#discussion_r818822849 This is the root cause of the problem, if you can fix this then it will be fine.

Reply:
Well, looks like my comment last night didn't successfully post out....

"Merge this first" is not a good practice in open source, from my experience, merged codes are rarely reviewed again and discussed, we don't have a deadline in open source so let's get things done completely (at least reach to an concensus).

The issue here to me is that the plugin includes all dependencies from all scopes, including test, provided, @asdf2014 did you try the option [`excludedScopes`](https://www.mojohaus.org/license-maven-plugin/add-third-party-mojo.html#excludedScopes) to exclude scopes `test`, `provided` dependencies? 

And because of this, I suspect that there are more unnecessary dependencies (`provided` scope) license added. Please overhaul these,

Reply:
@kezhenxu94 It is success refer to https://github.com/apache/incubator-seatunnel/pull/1387#discussion_r818822849  and I responded refer to https://github.com/apache/incubator-seatunnel/pull/1387#discussion_r818866682  😂

Reply:
@kezhenxu94 It was successful refer to https://github.com/apache/incubator-seatunnel/pull/1387#discussion_r818822849  and I responded refer to https://github.com/apache/incubator-seatunnel/pull/1387#discussion_r818866682  😂

Reply:
> @kezhenxu94 It was successful refer to [#1387 (comment)](https://github.com/apache/incubator-seatunnel/pull/1387#discussion_r818822849) and I responded refer to [#1387 (comment)](https://github.com/apache/incubator-seatunnel/pull/1387#discussion_r818866682) 😂

🤣 OK, the conversation is marked as resolved and I didn't notice this morning